### ISSUE TITLE
About training data generation

### ISSUE THREAD

**Issue by wdhudiekou** [lz118/Deep-Correlation-Network#1]
Thank you for great your work!

I have a question about your datasets.

I want to know how your training data is generated. The corresponding code is "RandomSpatialTransformation" in transform.py?

I'm looking forward to your reply. Thanks very much!

**Comment by lz118** [comment]
> Thank you for great your work!
> 
> I have a question about your datasets.
> 
> I want to know how your training data is generated. The corresponding code is "RandomSpatialTransformation" in transform.py?
> 
> I'm looking forward to your reply. Thanks very much!

Yes, we use "RandomSpatialTransformation" in transform.py to generate the pre-arranged training data.  And for convenience, we provide the unaligned dataset("VT5000-Train_unalign").
It is also a feasible manner using the "RandomSpatialTransformation" during training process.

**Comment by CSer-Tang-hao** [comment]
> > Thank you for great your work!
> > I have a question about your datasets.
> > I want to know how your training data is generated. The corresponding code is "RandomSpatialTransformation" in transform.py?
> > I'm looking forward to your reply. Thanks very much!
> 
> Yes, we use "RandomSpatialTransformation" in transform.py to generate the pre-arranged training data. And for convenience, we provide the unaligned dataset("VT5000-Train_unalign"). It is also a feasible manner using the "RandomSpatialTransformation" during training process.

I'm also confused about the data generation process. When using "RandomSpatialTransformation" to generate the pre-arranged training data, I find the degree of misalignment of the generated data is very large, but the degree of misalignment of your provided dataset is relatively small. Can you provide the optimal parameter settings when using "RandomSpatialTransformation" to generate provided unaligned dataset?

**Comment by lz118** [comment]
> > > Thank you for great your work!
> > > I have a question about your datasets.
> > > I want to know how your training data is generated. The corresponding code is "RandomSpatialTransformation" in transform.py?
> > > I'm looking forward to your reply. Thanks very much!
> > 
> > 
> > Yes, we use "RandomSpatialTransformation" in transform.py to generate the pre-arranged training data. And for convenience, we provide the unaligned dataset("VT5000-Train_unalign"). It is also a feasible manner using the "RandomSpatialTransformation" during training process.
> 
> I'm also confused about the data generation process. When using "RandomSpatialTransformation" to generate the pre-arranged training data, I find the degree of misalignment of the generated data is very large, but the degree of misalignment of your provided dataset is relatively small. Can you provide the optimal parameter settings when using "RandomSpatialTransformation" to generate provided unaligned dataset?

The degree of misalignment of the generated data depends on the intensity of the transformation. We set a  parameter named RTI to control the intensity.  It is a random value in provided code. We have not tried to set an optimal parameter. Therefore the samples in provided dataset have different spatial deviations.  If you want to get the samples with small spatial deviations, you can try to adjust RTI.
